control/vehicle_cmd_gate/README.md

```diff
@@ -0,0 +1,62 @@
+| `/input/auto/turn_indicators_cmd`          | `autoware_auto_vehicle_msgs::msg::TurnIndicatorsCommand`   | turn indicators command from planning module          |
+| `/input/auto/hazard_lights_cmd`            | `autoware_auto_vehicle_msgs::msg::HazardLightsCommand`     | hazard lights command from planning module            |
+| `/input/auto/gear_cmd`                     | `autoware_auto_vehicle_msgs::msg::GearCommand`             | gear command from planning module                     |
+| `/input/external/control_cmd`              | `autoware_auto_control_msgs::msg::AckermannControlCommand` | control command from external                         |
+| `/input/external/turn_indicators_cmd`      | `autoware_auto_vehicle_msgs::msg::TurnIndicatorsCommand`   | turn indicators command from external                 |
+| `/input/external/hazard_lights_cmd`        | `autoware_auto_vehicle_msgs::msg::HazardLightsCommand`     | hazard lights command from external                   |
+| `/input/external/gear_cmd`                 | `autoware_auto_vehicle_msgs::msg::GearCommand`             | gear command from external                            |
+| `/input/external_emergency_stop_heartbeat` | `autoware_external_api_msgs::msg::Heartbeat`               | heartbeat                                             |
+| `/input/gate_mode`                         | `autoware_control_msgs::msg::GateMode`                     | gate mode (AUTO or EXTERNAL)                          |
+| `/input/emergency_state`                   | `autoware_auto_system_msgs::msg::EmergencyState`           | used to detect the emergency situation of the vehicle |
+| `/input/emergency/control_cmd`             | `autoware_auto_control_msgs::msg::AckermannControlCommand` | control command from emergency handler                |
+| `/input/emergency/hazard_lights_cmd`       | `autoware_auto_vehicle_msgs::msg::HazardLightsCommand`     | hazard lights command from emergency handler          |
+| `/input/emergency/gear_cmd`                | `autoware_auto_vehicle_msgs::msg::GearCommand`             | gear command from emergency handler                   |
+| `/input/engage`                            | `autoware_auto_vehicle_msgs::msg::Engage`                  | engage signal                                         |
+
+### Output
+
+| Name                                  | Type                                                       | Description                                             |
+| ------------------------------------- | ---------------------------------------------------------- | ------------------------------------------------------- |
+| `/output/vehicle_cmd_emergency`       | `autoware_auto_system_msgs::msg::EmergencyState`           | emergency state which was originally in vehicle command |
+| `/output/command/control_cmd`         | `autoware_auto_control_msgs::msg::AckermannControlCommand` | gear command to vehicle                                 |
```

Comment:
lateral and longitudinal velocity ?

Reply:
thanks!
changed `control command` to `command for lateral and longitudinal velocity`!